caps/management/commands/import_filter_data.py

```diff
@@ -0,0 +1,69 @@
+import os
+import pandas as pd
+
+import requests
+import urllib3
+
+import ssl
+
+ssl._create_default_https_context = ssl._create_unverified_context
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+from django.core.management.base import BaseCommand, CommandError
+from django.conf import settings
+
+from os.path import join
+
+from caps.models import Council, PlanDocument
+
+FILTER_DATA_URL = "https://raw.githubusercontent.com/mysociety/climate_league_filter_datasets/main/data/outputs/all_filters.csv"
+FILTER_CSV = join(settings.DATA_DIR, "all_filters.csv")
+
+
+def get_data():
+    token = os.environ.get("PERSONAL_ACCESS_TOKEN", None)
+    headers = {"Authorization": "token " + token}
+    r = requests.get(FILTER_DATA_URL, headers=headers)
+    with open(FILTER_CSV, "wb") as outfile:
+        outfile.write(r.content)
+
+
+def import_filters():
+    ruc_lookup = dict((desc, code) for code, desc in Council.RUC_TYPES)
```

Comment:
Doesn't matter, but any reason not to do `{desc: code for code, desc in Council.RUC_TYPES}` here?

Reply:
no reason at all. Is there a general preference for one over the other?

Reply:
not really, I think technically there's an efficiency thing because you're not making a separate object first, but it doesn't matter at this size.